bellows/zigbee/zcl/foundation.py

```diff
@@ -101,7 +101,6 @@ class Discrete:
 DATA_TYPE_IDX = {
     t: tidx
     for tidx, (tname, t, ad) in DATA_TYPES.items()
-    if ad is Analog
```

Comment:
The bit of the spec I was thinking of is:

"2.4.7.1.7 Reportable Change Field
The reportable change field shall contain the minimum change to the attribute that
will result in a report being issued. This field is of variable length. For attributes
with 'analog' data type (see Table 2.16) the field has the same data type as the
attribute. The sign (if any) of the reportable change field is ignored.
For attributes of 'discrete' data type (see Table 2.16) this field is omitted."

It seems that I kinda handled that all properly elsewhere.

I think the problem with this change is that we now run into the problem that line 106 here tries (poorly) to fix -- DATA_TYPES is a many-to-one mapping, so now it's undetermined what, for example, t.uint64_t maps to - is it 0x19, 0x21, 0x31, 0xe8, or 0xe9.

What types in particular were you making this change for? Maybe the answer is to handle them like I've handled t.EmberEUI64?